### PR TITLE
[Doc] - Guide for sending logs to Kinesis

### PR DESCRIPTION
**Pull request by ddragosd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,107 @@ Backend systems
...
+    end
+
+    -- create a new instance of the logger
+    return logger_factory:getLogger(name, logger_module, logger_opts)
+end
+
+ngx.get_logger = get_logger
+```
+
+Then in the `log_by_lua` configure the logger send the information:
```
Comment (selfxp) [inline README.md]: @ddragosd I would update 

> configure the logger to send the information:

 to 

> configure the logger to start sending data:

File: README.md [hunk 2]
```diff
@@ -94,7 +94,107 @@ Backend systems
...
+    provider = "api-gateway.aws.AWSSTSCredentials",
+    role_ARN = "arn:aws:iam::" .. ngx.var.kinesis_aws_account .. ":role/" .. ngx.var.kinesis_iam_role,
+    role_session_name = "kinesis-logger-session",
+    shared_cache_dict = "aws_credentials"  -- dict for caching STS and IAM credentials
+}
+```
+
+Make sure to also configure the NGINX variables:
+
+* `kinesis_aws_account` - the AWS Account where the kinesis stream is
```
Comment (selfxp) [inline README.md]: @ddragosd I would update 

> `kinesis_aws_account` - the AWS Account where the kinesis stream is

to

> `kinesis_aws_account` - the AWS Account where the kinesis stream is configured

File: README.md [hunk 3]
```diff
@@ -94,7 +94,107 @@ Backend systems
...
+* `kinesis_iam_role` - the role to be assumed in order to send the logs to kinesis
+>INFO: If you send the logs into the same account where NGINX runs you don't need to configure any STS Credentials, but you can use IAM Credentials.
```
Comment (selfxp) [inline README.md]: @dragos Please indent the `INFO:` correctly

File: README.md [hunk 4]
```diff
@@ -94,7 +94,107 @@ Backend systems
...
+Make sure to also configure the NGINX variables:
+
+* `kinesis_aws_account` - the AWS Account where the kinesis stream is
+* `kinesis_iam_role` - the role to be assumed in order to send the logs to kinesis
+>INFO: If you send the logs into the same account where NGINX runs you don't need to configure any STS Credentials, but you can use IAM Credentials. 
+
+For more information about `AWSSTSCredentials` configuration see [the documentation](https://github.com/adobe-apiplatform/api-gateway-aws#sts-credentials).
+
+If you can't use IAM Credentials nor STS Credentials, then you can still send logs to Kinesis by configuring AWS with static `access_key` and `secret`.
+ These are not as secure as IAM/STS but work for non-AWS deployments:
```
Comment (selfxp) [inline README.md]: @ddragosd I would change 

> These are not as secure as IAM/STS but work for non-AWS deployments:

into 

> These are not as secure as IAM/STS but are working for non-AWS deployments:

File: README.md (hunk 3 above)
Comment (ddragosd) [inline README.md]: done

File: README.md (hunk 4 above)
Comment (ddragosd) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (ddragosd) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (ddragosd) [inline README.md]: done

**Pull request merged**